draft/Contest_API.md

```diff
@@ -349,26 +349,24 @@ a change in a single object (submission, judgement, language, team,
 etc.) or full endpoint. The general format for events is:
 
 ```json
-{"contest_id": "<id>", "endpoint": "<endpoint>", "id": "<id>", "data": <JSON data for element> }
+{"type": "<type>", "id": "<id>", "data": <JSON data for element> }
 ```
 
 | Name        | Type   | Required? | Nullable? | Description
 | :---------- | :----- | :-------- | :-------- | :----------
-| contest\_id | string | yes       | no        | The contest id.
-| endpoint    | string | yes       | yes       | The API endpoint, i.e. type of contest object above. Can be used for filtering.
-| id          | string | yes       | yes       | The id of the object that changed.
-| data        | object | yes       | yes       | The data is the object that would be returned if calling the corresponding API endpoint at this time, i.e. an object or null for deletions.
+| type        | string | yes       | yes       | The type of contest object that changed. Can be used for filtering.
```

Comment:
If we change from `endpoint` to `type` we need the type to endpoint mapping table (again).

Reply:
Or a place where we say that the type maps to the part after `/contest/<id>` except for `contest`?

Reply:
Updated as per discussion today.